### PR TITLE
cicd: update elastic beanstalk commands

### LINKED ISSUE #238: Upgrade AWS dev instance to python 3.11

### PR DESCRIPTION
**Pull request by korikuzma**
Closes #238 . Will open once #279 is merged

* switch to using python 3.11, cool-seq-tool, and p7zip

### CODE REVIEW

**Review by jsstevenson — CHANGES_REQUESTED** [review]

File: .ebextensions/01_app_config.config [hunk 1]
```diff
@@ -3,25 +3,25 @@ commands:
...
 
     02_s3_download:
         test: test ! -d "/usr/local/share/seqrepo"
         command: "aws s3 cp s3://${AWS_BUCKET_NAME}/${AWS_SEQREPO_OBJECT} /usr/local/share/seqrepo.zip --region us-east-2"
 
-    03_unzip_seqrepo:
+    03_p7zip_seqrepo:
         test: test -f "/usr/local/share/seqrepo.zip"
-        command: "unzip /usr/local/share/seqrepo.zip -d /usr/local/share"
+        command: "7za x /usr/local/share/seqrepo.zip -o/usr/local/share -y"
```
[the remark this answers is not in the document]
Comment (korikuzma) [reply .ebextensions/01_app_config.config:24]: @jsstevenson Your suggestion is not the right syntax

[the remark this answers is not in the document]
Comment (korikuzma) [reply .ebextensions/01_app_config.config:24]: https://www.mankier.com/1/7za

[the remark this answers is not in the document]
Comment (jsstevenson) [reply .ebextensions/01_app_config.config:24]: Well now I'm bugging even more

**Review by jsstevenson — APPROVED** [review]

File: .ebextensions/01_app_config.config (hunk 1 above)
[the remark this answers is not in the document]
Comment (korikuzma) [reply .ebextensions/01_app_config.config:24]: Lol when I initially updated in the other repos, I made this change and it failed. So you're not alone

**Pull request merged**